api/sessionrecording/session_recording_test.go

```diff
@@ -0,0 +1,31 @@
+package sessionrecording_test
```

Comment:
The tests are pretty light within `api/sessionrecording`, but looks like `lib/events` is testing most of `Reader`. I moved the only test case that seemed to be testing `Reader` directly. Everything else looks like it's using `Reader` to verify writes are as expected.